tools/backup_analysis.py

```diff
@@ -93,8 +93,7 @@ def older_dir(dir1, dir2):
 # is kept to analyze the state of the system.
 def compute_type(filename, filemeta):
     # Figure out the type of file it is based on the name and metadata.
-    # For collections and indexes, if logging is disabled, then they are replicated.
-    #   if logging is enabled on an index, it is a local table.
+    # For collections if logging is disabled, then it is replicated.
```

Comment:
Change "they is replicated" to "they are replicated" since collections is plural.